.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,33 @@
+name: Versioning
+
+on:
+  pull_request:
+
+jobs:
+  bump-version:
+    name: Check version has been bumped
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout to BASE
+        uses: actions/checkout@v4
+        with:
+          ref: dev
+
+      - name: Get BASE_VERSION
+        run: |
+          echo "BASE_VERSION=$(grep '"version"' apps/web/package.json | sed -E 's/.*"version": *"([^"]+)".*/\1/')" >> $GITHUB_ENV
```

Comment:
If we can put the path in a variable, that'd be great (even if it's just in an env declared in the workflow)